common/build.gradle

```diff
@@ -0,0 +1,96 @@
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform'
+    id 'org.jetbrains.kotlin.native.cocoapods'
+    id 'com.android.library'
+}
+
+group 'com.conferenceplatform'
+version '0.0.1' // CocoaPods requires the podspec to have a version.
+
+repositories {
+    mavenCentral()
+}
+
+android {
+    compileSdkVersion(29)
```

Comment:
was planning to add and consolidate constants in a followup PR for things like kotlin version, sdk version, dependencies, etc